src/components/IconLayer.tsx

```diff
@@ -14,13 +15,13 @@ type IconLayerProps = {
 const IconLayer: React.FC<IconLayerProps> = (props) => {
   const iconProps = { style: props.style };
   switch(props.type) {
-    case "fill-extrusion": return <BsDiamondFill {...iconProps} />;
-    case "raster": return <BsDiamond {...iconProps} />;
+    case "fill-extrusion": return <MdOutlineVerticalShades {...iconProps} />;
+    case "raster": return <BsDiamondFill {...iconProps} />;
     case "hillshade": return <BsSun {...iconProps} />;
-    case "color-relief": return <MdBubbleChart {...iconProps} />;
-    case "heatmap": return <BsDiamond {...iconProps} />;
-    case "fill": return <BsDiamond {...iconProps} />;
-    case "background": return <BsDiamondFill {...iconProps} />;
+    case "color-relief": return <CiMountain1 {...iconProps} />;
+    case "heatmap": return <MdBubbleChart {...iconProps} />;
+    case "fill": return <BsDiamondFill {...iconProps} />;
+    case "background": return <MdOutlineStreetview {...iconProps} />;
```

Comment:
MdOutlineStreetview seems a bit strange for background.
At least I don't consider this very backgroundy 😉

<img width="106" height="107" alt="image" src="https://github.com/user-attachments/assets/9d91791a-f567-4ebb-be8f-bd3a580e9367" />

How about 

<img width="103" height="97" alt="image" src="https://github.com/user-attachments/assets/b6733235-de8a-4af6-a8d1-5caef0dcffc0" />